src/authMappingUtils.js

```diff
@@ -76,3 +76,17 @@ export const userHasMethodOnAnyProject = (method, userAuthMapping = {}) => {
   const hasMethod = actionArrays.some(x => x.some(actionHasMethod));
   return hasMethod;
 };
+
+export const shouldDisplaySubmissionUIComponents = (project, userAuthMapping, useArboristUI) => {
+  if (
+    !useArboristUI
+    || (isRootUrl(project) && userHasSheepdogProgramAdmin(userAuthMapping))
+    || (isProgramUrl(project) && userHasSheepdogProjectAdmin(userAuthMapping))
+    || userHasMethodOnProject('create', project, userAuthMapping)
+    || userHasMethodOnProject('update', project, userAuthMapping)
```

Comment:
lets also add `userHasMethodOnProject('read', project, userAuthMapping)` in here. If user has `read` premission then the user shuold see the nodes but connot submit